event/events.go

```diff
@@ -38,6 +39,7 @@ func ToEvent(op types.Operation, blockID string, blockNum uint32, timestamp time
 	return commonEvent
 }
 
+// Common Event
```

Comment:
why do we need those comments

Reply:
Simply for navigation

Reply:
So if we are leaving such comments maybe we can follow Golint format for them? `CommonEvent` instead `Common Event`

Reply:
Good point, done